### PR TITLE
Task/add from log

### PR DESCRIPTION
**Pull request by AlvaroVega**
issue https://github.com/telefonicaid/iotagent-node-lib/issues/918

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Add: include from in log context (#918)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: Do we have some document in this repo describing the log system? In positive case, it should be updated due to ths PR. In negative case, this is NTC.

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:1]: Not sure you mean something like this: https://github.com/telefonicaid/iotagent-node-lib/blob/96e12e12847dda7afd8cf6fb7af243a0ca9970c2/doc/operations.md#logs

Comment (fgalan) [reply CHANGES_NEXT_RELEASE:1]: NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**